build.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+### VARIABLES
+
+DOCKER_IMAGE='proot/proot-buildenv:latest'
+
+CONTAINER_USERNAME='dummy'
+CONTAINER_GROUPNAME='dummy'
+GROUP_ID=$(id -g)
+USER_ID=$(id -u)
+
+# must be an absolute path
+TARGET_DIR=${TARGET_DIR:-"$(pwd)/target"}
+
+# VOLUMES must be formatted as Docker expects them -> /path_on_host:/mountpoint
+# specify multiple volumes using the ; separator
+VOLUMES=${VOLUMES:-""}
+
+### FUNCTIONS
+
+create_user_cmd()
+{
+  if [[ ${USER_ID} -ne 0 ]]; then
+    echo \
+      groupadd -f -g ${GROUP_ID} ${CONTAINER_GROUPNAME} '&&' \
```

Comment:
Is there a reason to create the user during the execution of the image?
I guess we could create a default user in Dockerfile and use it for for building the static binaries.

This can potentially things a bit as you do not need to create a user for each user that attempts to build the binaries. WDYT?

Reply:
fixed i got rid of it :)